test/GraphClient/constructor.test.ts

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+import { assert } from  'chai'
+import GraphClient from './../../src/GraphClient'
+
+describe('GraphClient.constructor()', () => {
+  let subgraphEndPoint;
+
+  beforeEach(() => {
+    subgraphEndPoint = '/subgraph/v1/subgraph1';
+  });
+
+  it('should construct with correct parameters', async () => {
+    const graphClient = new GraphClient(subgraphEndPoint);
+    assert(graphClient);
```

Comment:
Assert `subgraphEndPoint` is set or not.

Reply:
subgraphEndPoint is not a property of GraphClient in the updated code.